velox/type/Timestamp.h

```diff
@@ -34,8 +35,20 @@ struct Timestamp {
  public:
   enum class Precision : int { kMilliseconds = 3, kNanoseconds = 9 };
   constexpr Timestamp() : seconds_(0), nanos_(0) {}
-  constexpr Timestamp(int64_t seconds, uint64_t nanos)
-      : seconds_(seconds), nanos_(nanos) {}
+  Timestamp(int64_t seconds, uint64_t nanos) {
+    VELOX_CHECK_GE(
+        seconds,
+        dwio::common::MIN_SECONDS,
```

Comment:
We should not use 'dwio' cosntants outside of the DWRF reader. See if other constants available or add new ones.

Reply:
After consideration, I decided to remove the verification of the range of seconds. Nanos needs to be less than 10^9, which is the limit required by the Timestamp type itself, but the range of seconds depends on the ORC format.

https://github.com/apache/orc/blob/ec2ea9c6aff8b8515452df651f08695639c18cbb/java/core/src/java/org/apache/orc/impl/writer/TimestampTreeWriter.java#LL41C5-L41C5

 I create constant of nanos max value in Timestamp.

Reply:
> I decided to remove the verification of the range of seconds. Nanos needs to be less than 10^9, which is the limit required by the Timestamp type itself, but the range of seconds depends on the ORC format.

Makes sense to me. Thanks.